### ISSUE TITLE
Spore Spawn soft lock

### ISSUE THREAD

**Issue by beauxq** [Dessyreqt/smrandomizer#33]
Do you try to avoid soft locks with this?

If you go to Spore Spawn without super missiles, and then you don't get super missiles after him, you are locked in that room.

A possible solution could be:
```
if (the item behind Spore Spawn is not super missiles) {
    make the door to get to Spore Spawn green;
}
```

**Comment by moriwizz** [comment]
Yes, there are a few softlocks that you need to avoid. I think nobody noticed this particular issue because they always get Spore Spawn's item without fighting him.
